topics/single-cell/tutorials/bulk-deconvolution-evaluate/tutorial.md

```diff
@@ -0,0 +1,585 @@
+
+<iframe title="Galaxy Workflow Embed" style="width: 100%; height: 700px; border: none;" src="https://usegalaxy.eu/published/workflow?id=cb27f805d076ee9f&embed=true&buttons=true&about=false&heading=false&minimap=true&zoom_controls=true&initialX=-20&initialY=-20&zoom=0.5"></iframe>
+
+The output of this workflow will be the psuedo-bulk and actual cell proportions for both samples A and B. If you inspect one of the elements in the `Actual Cell Proportions` collection, you should see a table similar to the following:
+
+|                         | A_actual   |
+|-------------------------|------------|
+| acinar                  | 0.090000   |
+| alpha                   | 0.415000   |
+| beta                    | 0.170000   |
+| co-expression           | 0.050000   |
+| delta                   | 0.070000   |
+| ductal                  | 0.105000   |
+| endothelial             | 0.015000   |
+| gamma                   | 0.050000   |
+| mast                    | 0.010000   |
+| unclassified endocrine  | 0.025000   |
+
+Comparing the above table with the cell-type counts of the original single-cell data, does this look correct? Well the top 3 cell-types with the highest proportion in the single-cell data are: alpha, beta, ductal. Which aligns with the proportion values of the above data! There may be some variance due to the randomly selected cells. Also note that some of the lesser common cell types (like `MHC class II`) aren't present in the above table, again this is due to the 200 randomly selected cells for this specific sample and isn't of concern.
+
+# Perform deconvolution on the pseudo-bulk data
+
+Now that we have our pseudo-bulk data alongside the actual proportion values. Our next step is to run deconvolution to get predicted cell-type proportions! Currently, Galaxy contains two tools for performing deconvolution: **MuSiC** and **NNLS**. We will use both of these tools in this tutorial and compare their results together.
+
+The following workflow will take the two pseudo-bulk samples (A and B), as well as the original single-cell data as reference and output the deconvolution results for both samples and deconvolution methods. Thus producing 4 output collections. The pdf results of the deconvolution tools will also be outputted from the workflow but won't be needed for the tutorial.
+
+> <hands-on-title>Run inferring cellular proportions workflow</hands-on-title>
+>
+> 1. **Import the workflow** into Galaxy
+>    - Copy the URL (e.g. via right-click) of [this workflow](https://usegalaxy.eu/u/hexhowells/w/deconv-eval-stage-2) or download it to your computer.
+>    - Import the workflow into Galaxy
+>
+>    {% snippet faqs/galaxy/workflows_run_trs.md path="topics/transcriptomics/tutorials/rna-seq-reads-to-counts/workflows/qc_report.ga" title="QC Report" %}
+>
+> 2. Run **Workflow inferring cellular proportions** {% icon workflow %} using the following parameters:
+>    - {% icon param-collection %} *"Pseudobulk - A"*: `expression data - A`

```

Comment:
These collections also did not appear in the drop down menus, and rather had to be dragged and dropped.